engine/install/binaries.md

```diff
@@ -19,13 +21,14 @@ system's package management system to manage Docker installation and upgrades.
 Be aware that 32-bit static binary archives do not include the Docker daemon.
```

Comment:
> Be aware that 32-bit static binary archives do not include the Docker daemon.

Looks like this is no longer the case (32-bit tars also include the daemon and containerd)